.circleci/config.yml

```diff
@@ -385,7 +390,7 @@ jobs:
           condition:
             not:
               matches:
-                pattern: /^master$/
+                pattern: /^master|stable-v12\.12\.x$/
```

Comment:
I tested this Regex using https://regex101.com using Java 8 settings. Apparently CircleCI uses Java style regular expressions.